packages/angular_devkit/schematics/tools/index.ts

```diff
@@ -6,6 +6,7 @@
  * found in the LICENSE file at https://angular.io/license
  */
 export * from './description';
+export * from './export-ref';
```

Comment:
This change to the public API should be a feat in it's own commit.

Reply:
`ExportRef` should be part of core anyway.

Reply:
It's in a feature. This was from Mike's PR so I don't want to refactor it. I'll refactor it later (would like to move it out of schematics).